std/file.d

```diff
@@ -862,7 +862,7 @@ unittest
 version(StdDdoc) void getTimesWin(R)(R name,
                                   out SysTime fileCreationTime,
                                   out SysTime fileAccessTime,
-                                  out SysTime fileModificationTime)
+                                  out SysTime fileModificationTime) @safe
```

Comment:
I guess it's inferred? Otherwise, how would it compile when `timeLastModified` is explicitly `@safe`, and it calls `getTimesWin`?